package.json

```diff
@@ -21,9 +21,11 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-games#readme",
   "dependencies": {
-    "pg-format": "^1.0.4",
     "dotenv": "^16.0.0",
-    "pg": "^8.7.3"
+    "express": "^4.18.1",
+    "pg": "^8.7.3",
+    "pg-format": "^1.0.4",
+    "supertest": "^6.2.4"
```

Comment:
supertest is a dev dependency as it is not need for the app to run in a production environment, just move this down to the devDependencies section of this file